### ISSUE TITLE
Links to RTD do not work

### ISSUE THREAD

**Issue by frankwiles** [GaretJax/django-click#5]
Links in README and on PyPI link to ReadTheDocs, but it 404s.

**Comment by frankwiles** [comment]
Oh I see in the README this is listed as a TODO, but might want to just remove them to avoid confusion.

**Comment by GaretJax** [comment]
You're right, thanks for reporting it. One of the things I wanted to do right away but then real life came in the way. :-)

**Comment by frankwiles** [comment]
COMPLETELY understand! (says the guy with half a dozen PRs waiting on him)

**Comment by FlipperPA** [comment]
@frankwiles @GaretJax I've got a work in progress for docs that is coming along slowly. https://github.com/FlipperPA/django-click/tree/docs-2.0
